source/beamline/SXRMB/SXRMBBeamline.cpp

```diff
@@ -642,19 +642,22 @@ void SXRMBBeamline::setupSampleStage()
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageZ_);
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageR_);
 
-	// Ambiant Endstation sample state
+	// Ambiant Endstation sample stage
+	ambiantTableHeight_ = new AMPVwStatusControl("Ambiant Table Height", "TBL0000-E07-01:Y:mm:fbk", "TBL0000-E07-01:Y:mm", "TBL0000-E07-01:Y:mm:status", QString(), this, 0.005, 2.0, new CLSMAXvControlStatusChecker());
```

Comment:
@dretrex this is a pseudo motor, which doesn't have a stop PV. The real table motors have the stop PV. 

I will ask Tonia about it.